
>>> START RUNNING: FedAVG-Equal-Non-IID-01-2.1.1 - Train mode: benchmark - Dataset: chexpert
Traceback (most recent call last):
  File "train.py", line 343, in <module>
    main(args)
  File "train.py", line 126, in main
    train_dataset, test_dataset, list_idx_sample, valid_set = load_dataset(args.dataset_name, args.path_data_idx)
  File "train.py", line 88, in load_dataset
    dataset, train_dataset, valid_set, test_dataset = load_cheXpert_dataset()
  File "/mnt/disk1/dungnt/federated-learning-dqn/utils/loader.py", line 306, in load_cheXpert_dataset
    dataset = CheXpertDataSet(pathFileTrain ,transformSequence, policy="ones")
  File "/mnt/disk1/dungnt/federated-learning-dqn/utils/loader.py", line 225, in __init__
    csvReader = csv.reader(f)
NameError: name 'csv' is not defined